gpytorch/lazy/added_diag_lazy_tensor.py

```diff
@@ -88,7 +88,7 @@ def _preconditioner(self):
             return self.preconditioner_override(self)
```

Comment:
These changes are probably breaking for anyone who was using preconditioner_override before. @wjmaddox

Reply:
I don't think I have anything in the works that's actually using the preconditioner_override. But, is this necessary?

Reply:
I reverted back to the original preconditioner return values (preconditioner_function, precond_lt, logdet_p) to address @jacobrgardner 's other comments. Does this resolve this issue as well?

Reply:
Yes, preconditioner_override should work fine again with the revert